SECURITY.md

```diff
@@ -0,0 +1,12 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+Please read the guidelines [on the
+official website](https://www.python.org/dev/security/) for
+instructions on how to report a security issues to
+the Python team responsibly.
+
+To reach the response team, email 
+<a href="mailto:%73%65%63%75%72%69%74%79%40%70%79%74%68%6F%6E%2E%6F%72%67">security
```

Comment:
Do we really need to obfuscate this?

Reply:
I personally don't mind, my suggestion was to copy https://www.python.org/dev/security/, there must have been a reason to do it there 🤷